src/move.c

```diff
@@ -3215,10 +3215,10 @@ static int scroll_with_sms(int dir, long count, long *curscount)
 
 	int width1 = curwin->w_width - curwin_col_off();
 	int width2 = width1 + curwin_col_off2();
-	count = 1 + (curwin->w_skipcol - width1) / width2;
+	count = 1 + ceil((float)(curwin->w_skipcol - width1) / width2);
```

Comment:
```suggestion
	count = 1 + (curwin->w_skipcol - width1 + width2 - 1) / width2;
```

Reply:
Yeah I used that first but thought this was more self-documenting. Should I add a comment for this form?